action.yml

```diff
@@ -17,7 +17,7 @@ inputs:
     required: false
   COMMIT_MSG:
     description: "Commit message used while committing to the repo"
-    default: ":zap: Update README with the recent activity"
+    default: ":zap: chore(readme): Update README with the recent activity"
```

Comment:
Is this change necessary?

Reply:
Just wanted to follow the standard commit practice. Will revert it.

Reply:
Done